Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Curtains.Domain.Models
+{
+    internal class ReviewMapping : IEntityTypeConfiguration<ReviewModel>
+    {
+        public void Configure(EntityTypeBuilder<ReviewModel> builder)
+        {
+            builder.ToTable(nameof(ReviewModel));
+
+            builder.Property(r => r.PersonName)
+                .HasMaxLength(10)
+                .UseCollation("SQL_Latin1_General_CP1_CI_AS");
```

Comment:
How we can simplify using same collations instead of using it in every property?

Reply:
May be it's make sense to move it into appsettings.json
Or we can create constants static class and use them from it.